### PR TITLE
docs: add FAQ item about support of next.js app router

### PR DESCRIPTION
**Pull request by abenhamdine**
based on comments here : https://github.com/shuding/nextra/discussions/1142#discussioncomment-4552759
I put this entry first in FAQ list because it seems quite important.

### CODE REVIEW

File: docs/pages/docs/index.mdx [hunk 1]
```diff
@@ -47,6 +47,11 @@ export function FAQBox({ title, children }) {
   )
 }
 
+<FAQBox title="Can I use Nextra with Next.js app router?">
+  No, Nextra only works with the ./pages directory at the moment. Support for the app router has not yet been implemented and might be a v3 feature.
```
Comment (dimaMachina) [inline docs/pages/docs/index.mdx:51]: There are no plans for v3

Comment (abenhamdine) [reply docs/pages/docs/index.mdx:51]: ah ok I will remove it.

**Review by dimaMachina — APPROVED** [review]

**Comment by abenhamdine** [comment]
CI "analyze" is failing, IDKW : error messge is  "Resource not accessible by integration"

**Review by cruzluna — APPROVED** [review]
This is greatly needed in the docs.

File: docs/pages/docs/index.mdx [hunk 2]
```diff
@@ -47,6 +47,13 @@ export function FAQBox({ title, children }) {
   )
 }
 
+<FAQBox title="Can I use Nextra with Next.js app router?">
+  No, Nextra only works with the ./pages directory at the moment. Support for
```
Comment (dimaMachina) [inline docs/pages/docs/index.mdx:51]:
```suggestion
  No, Nextra only works with the `/pages` directory at the moment. Support for
```

File: docs/pages/docs/index.mdx [hunk 3]
```diff
@@ -47,6 +47,13 @@ export function FAQBox({ title, children }) {
...
+  the app router has not been implemented and there's currently no plan to add
+  it. But you can use app/ and pages/ at the same time : just put your docs
```
Comment (dimaMachina) [inline docs/pages/docs/index.mdx:53]:
```suggestion
  it. But you can use `/app` and `/pages` at the same time: just put your docs
```

File: docs/pages/docs/index.mdx [hunk 4]
```diff
@@ -47,6 +47,13 @@ export function FAQBox({ title, children }) {
   )
 }
 
+<FAQBox title="Can I use Nextra with Next.js app router?">
+  No, Nextra only works with the `/pages` directory at the moment. Support for
+  the app router has not been implemented and there's currently no plan to add
+  it. But you can use `/app` and `/pages` at the same time - just put your docs
+  inside pages/ and your other routes in app/
```
Comment (dimaMachina) [inline docs/pages/docs/index.mdx:54]:
```suggestion
  inside `/pages` and your other routes in `/app`
```

**Pull request merged**